src/config/syscheck-config.c

```diff
@@ -59,6 +59,77 @@ directory_t *fim_create_directory(const char *path,
     return new_entry;
 }
 
+int initialize_syscheck_configuration(syscheck_config *syscheck) {
+    if (syscheck == NULL) {
+        os_calloc(1, sizeof(syscheck_config), syscheck);
```

Comment:
This allocation is not freed at `Free_Syscheck`. Please add the `free` for it at the end of that function.